src/Microsoft.Identity.Web.TokenAcquisition.Abstractions/ApplicationOptions/CredentialDescription.cs

```diff
@@ -30,7 +30,7 @@ public class CredentialDescription
         /// this value is the path to the certificate in the cert store, for instance <c>CurrentUser/My</c>.</item>
         /// </list>
         /// </summary>
-        protected string? Container
+        public string? Container
```

Comment:
For our records:
We might revert these changes (protected to public) when we work on "Microsoft.Identity.Web.Credentials provide support for certificates, symmetric keys and signed assertions"